specs/altair/sync-protocol.md

```diff
@@ -47,34 +49,25 @@ uses sync committees introduced in [this beacon chain extension](./beacon-chain.
 
 ### Misc
 
-| Name | Value |
-| - | - |
-| `MIN_SYNC_COMMITTEE_PARTICIPANTS` | `1` |
+| Name | Value | Notes |
+| - | - | - |
+| `MIN_SYNC_COMMITTEE_PARTICIPANTS` | `1` | |
+| `UPDATE_TIMEOUT` | `SLOTS_PER_EPOCH * EPOCHS_PER_SYNC_COMMITTEE_PERIOD` | ~27.3 hours |
+| `SAFETY_THRESHOLD_PERIOD` | `SLOTS_PER_EPOCH * EPOCHS_PER_SYNC_COMMITTEE_PERIOD // 2` | ~13.6 hours |
 
 ## Containers
 
-### `LightClientSnapshot`
-
-```python
-class LightClientSnapshot(Container):
-    # Beacon block header
-    header: BeaconBlockHeader
-    # Sync committees corresponding to the header
-    current_sync_committee: SyncCommittee
-    next_sync_committee: SyncCommittee
-```
-
 ### `LightClientUpdate`
 
 ```python
 class LightClientUpdate(Container):
-    # Update beacon block header
-    header: BeaconBlockHeader
-    # Next sync committee corresponding to the header
+    # The beacon block header that is attested to by the sync committee
+    attested_header: BeaconBlockHeader
+    # Next sync committee corresponding to the active header
     next_sync_committee: SyncCommittee
     next_sync_committee_branch: Vector[Bytes32, floorlog2(NEXT_SYNC_COMMITTEE_INDEX)]
-    # Finality proof for the update header
-    finality_header: BeaconBlockHeader
+    # The finalized beacon block header attested to by Merkle branch
+    finalized_header: BeaconBlockHeader
     finality_branch: Vector[Bytes32, floorlog2(FINALIZED_ROOT_INDEX)]
     # Sync committee aggregate signature
     sync_committee_bits: Bitvector[SYNC_COMMITTEE_SIZE]
```

Comment:
I would compact this into sync_aggregate like in https://github.com/ethereum/consensus-specs/pull/2267/files#diff-e1b1ce81657edc3d8874cfec98e2f7d2f8d0c478cc8cf0998dd411b6da001ecbR116 makes the implementation simpler

Reply:
```
class SyncAggregate(Container):
    sync_committee_bits: Bitvector[SYNC_COMMITTEE_SIZE]
    sync_committee_signature: BLSSignature
```

Reply:
Wouldn't this make the serialized size of `LightClientUpdate` larger due to the extra nesting? I thought this was all about minimizing bytes. Granted, the change is minimal.

Reply:
@etan-status in ssz serialized form the size is the same, is json form it's negligibly larger